tests/storage/cdi_import/test_import_http.py

```diff
@@ -565,43 +565,30 @@ def test_vmi_image_size(
         cert_configmap=internal_http_configmap.name,
     ) as dv:
         dv.wait_for_dv_success(timeout=TIMEOUT_4MIN)
-        containers = get_containers_for_pods_with_pvc(
-            volume_mode=storage_class_matrix__module__[storage_class_name_scope_module]["volume_mode"], pvc_name=dv.name
-        )
+        containers = get_containers_for_pods_with_pvc(volume_mode=volume_mode, pvc_name=dv.name)
+
         with create_vm_from_dv(dv=dv, start=False):
             with PodWithPVC(
                 namespace=dv.namespace,
                 name=f"{dv.name}-pod",
                 pvc_name=dv.name,
                 containers=containers,
             ) as pod:
-                # In case of file system volume mode, the FS overhead should be taken into account
-                # the default overhead is 5.5%, so in order to reserve the 5.5% for the overhead
-                # the actual size for the disk will be smaller than the requested size
-                if dv.volume_mode == DataVolume.VolumeMode.FILE:
-                    size *= 1 - default_fs_overhead
-                    # In case that size < 1, convert from Gi to Mi
-                    if size < 1:
-                        size = GiB(size).to_MiB().value
-                        unit = m_byte
                 pod.wait_for_status(status=pod.Status.RUNNING)
-                virtual_size_output_line = pod.execute(
+                output = pod.execute(
                     command=[
                         "bash",
                         "-c",
-                        "qemu-img info /pvc/disk.img|grep 'virtual size'",
+                        "qemu-img info /pvc/disk.img | grep 'virtual size'",
                     ]
                 )
-                match = re.search(
-                    r":\s*(\d+)\s*([MG])",
-                    virtual_size_output_line,
-                )
-                assert match, (
-                    "Incorrect virtual size found on disk image /pvc/disk.img\n"
-                    f"Virtual size reported as: {virtual_size_output_line}"
+                match = re.search(r"\((\d+)\s+bytes\)", output)
+                assert match, f"Failed to extract byte size from disk image info\nOutput: {output}"
+                image_bytes = int(match.group(1))
+
+                assert math.isclose(image_bytes, expected_virtual_size_bytes, rel_tol=default_fs_overhead), (
```

Comment:
fs overhead only plays a role for filesystem storage, so we shouldn't consider it for Block